### DISCUSSION TITLE
Does this lib supports Full text Search

### DISCUSSION THREAD

**Discussion by AathifMahir** [supabase-community/supabase-csharp#137] [Q&A] [answered]
Hello Folks,

I can't seems to be find any docs related to full text Search on c# when it comes to Supabase, therefore I'm here confirm whether this lib supports full text Search of supabase

**Comment by acupofjose** [comment, marked as answer]
Yes it can - there are a couple of options for the full text config, but the postgrest-csharp lib supports the requests. You can find the docs here: https://supabase.com/docs/reference/csharp/match

Also, the relevant JS docs (just for reference, as this library essentially ports the JS library) are here: https://supabase.com/docs/reference/javascript/textsearch

**Comment by AathifMahir** [comment]
> Yes it can - there are a couple of options for the full text config, but the postgrest-csharp lib supports the requests. You can find the docs here: https://supabase.com/docs/reference/csharp/match
> 
> Also, the relevant JS docs (just for reference, as this library essentially ports the JS library) are here: https://supabase.com/docs/reference/javascript/textsearch

That's kind a answers it, Thanks
